### PR TITLE
Cleanup of doom.cpp

### PR DESCRIPTION
**Pull request by vlolteanu**

### CODE REVIEW

File: Source/doom.cpp [hunk 1]
```diff
@@ -44,48 +42,32 @@ int doom_get_frame_from_time()
...
+	} else {
+		doom_close();
 	}
 }
 
 void doom_close()
 {
 	doomflag = false;
-	doom_cleanup();
+	doomCel.reset();
```
Comment (glebm) [inline Source/doom.cpp:70]: nit: I find that this reads a bit better:

```suggestion
	doomCel = nullptr;
```

**Review by glebm — DISMISSED** [review]

File: Source/doom.cpp [hunk 2]
```diff
@@ -11,27 +11,25 @@ namespace devilution {
 
 int doom_quest_time;
 int doom_stars_drawn;
-BYTE *pDoomCel;
+std::unique_ptr<uint8_t[]> doomCel;
```
Comment (AJenbo) [inline Source/doom.cpp:14]: Globals should be CamelCase

```suggestion
std::unique_ptr<uint8_t[]> DoomCel;
```

Comment (vlolteanu) [reply Source/doom.cpp]: Done

File: Source/doom.cpp [hunk 3]
```diff
@@ -9,10 +9,9 @@
...
 namespace devilution {
 
-int doom_quest_time;
-int doom_stars_drawn;
-std::unique_ptr<uint8_t[]> doomCel;
-bool doomflag;
+int DoomQuestTime;
+std::unique_ptr<uint8_t[]> DoomCel;
+bool DoomFlag;
 int DoomQuestState;
```
Comment (AJenbo) [inline Source/doom.cpp]: thanks, this will make the global cleanup seam less drastic :D

File: Source/doom.cpp (hunk 1 above)
Comment (AJenbo) [reply Source/doom.cpp]: would you apply this also?

Comment (vlolteanu) [reply Source/doom.cpp]: Sure, why not?

File: Source/doom.cpp (hunk 3 above)
Comment (vlolteanu) [reply Source/doom.cpp]: No problem.

**Pull request merged**